### PR TITLE
New charges

### PR DESCRIPTION
**Pull request by Blipz**

### CODE REVIEW

File: public/charges/arrowsSheaf.svg [hunk 1]
```diff
@@ -4,18 +4,21 @@
...
       <path d="M 99.164 134.714v -70.785s 0.363 -0.132 0.825 -0.132 0.825 0.209 0.825 0.209v 70.785z"/>
+      <ellipse cx="99.989" cy="135.077" rx="1.65" ry="0.913"/>
       <path class="secondary" d="M 101.023 135.11l -0.385 5.819 2.398 -4.895H 106.6l -6.6 7.909 -6.6 -7.909h 3.564l 2.398 4.895 -0.385 -5.819s 0.308 -0.286 1.034 -0.286c 0.704 0.0 1.012 0.286 1.012 0.286z"/>
       <path fill="#000" stroke="none" d="M 100.968 77.096l 2.035 -4.444 -2.332 4.301zm -0.011 -3.652l 2.024 -3.322 -2.299 3.135zm -0.011 -3.157l 2.112 -3.212 -2.376 3.014zm -1.628 6.666l -2.343 -4.301 2.046 4.444zm -0.011 -3.696l -2.299 -3.135 2.024 3.322zm 0.0 -3.168l -2.376 -3.014 2.101 3.212z"/>
     </g>
     <g transform="rotate(45.0 104.7289 101.481)">
       <path class="tertiary" d="M 100.814 66.47a 6.6 -6.6 0 0 1 2.453 -0.671c 1.056 0.0 2.222 1.056 2.222 2.288 0.0 3.399 -4.675 12.254 -4.675 12.254zm -1.65 0.0s -1.331 -0.671 -2.453 -0.671c -1.056 0.0 -2.222 1.056 -2.222 2.288 0.0 3.399 4.675 12.254 4.675 12.254z"/>
       <path d="M 99.164 134.714v -70.785s 0.363 -0.132 0.825 -0.132 0.825 0.209 0.825 0.209v 70.785z"/>
+      <ellipse cx="99.989" cy="135.077" rx="1.65" ry="0.913"/>
```
Comment (Azgaar) [inline public/charges/arrowsSheaf.svg:14]: I have deleted it before, we don't need it

**Review by Azgaar — APPROVED** [review]

File: public/charges/arrowsSheaf.svg (hunk 1 above)
Comment (Blipz) [reply public/charges/arrowsSheaf.svg:14]: These are visible ellipses near the arrowhead, `ry` was mistakenly converted to a negative value in d874d7c936f749190bea66d416eb2fde48c1afe8.

Comment (Azgaar) [reply public/charges/arrowsSheaf.svg:14]: ok

**Comment by Azgaar** [comment]
Would need to add the same to Armoria API and FMG. Can you do it? If not, it can wait

**Comment by Blipz** [comment]
> Would need to add the same to Armoria API and FMG. Can you do it? If not, it can wait

Yes, I'll look at that a bit later.

**Comment by Azgaar** [comment]
thanks

**Pull request merged**